### ISSUE TITLE
injectGlobal doesn't working in SSR

### ISSUE THREAD

**Issue by obetomuniz** [styled-components/styled-components#749]
## Version
2.0.0-15

## Expected Behavior

I have an injectGlobal style and I want to fetch this CSS on the server.

![screen shot 2017-04-27 at 1 43 25 am](https://cloud.githubusercontent.com/assets/1680157/25468243/f5427f18-2aea-11e7-81a0-35c782a0a503.png)

## Actual Behavior

I'm using `const styles = styleSheet.getCSS();` to fetch my current CSS on the server, but it just return the CSS of my components, global styles are not returned.

![screen shot 2017-04-27 at 1 38 23 am](https://cloud.githubusercontent.com/assets/1680157/25468222/bcdfac4a-2aea-11e7-96b0-bea705498ed9.png)

**Comment by erikras** [comment]
For what it's worth, using the ye olde
```jsx
const styles = (
  <style
    dangerouslySetInnerHTML={{
      __html: styleSheet.rules().map(rule => rule.cssText).join('\n')
    }}
  />
)
```
method works with `injectGlobal` in `2.0.0-15`.

**Comment by oscar-b** [comment]
Where is your call to `injectGlobal`?

**Comment by obetomuniz** [comment]
@erikras Nice tip, but sadly it did not work.

@oscar-b I'm using on my <Root /> component. The location where I put it matters? I thought that it can be inserted anywhere.

**Comment by oscar-b** [comment]
@obetomuniz In what method of your Root-component?

**Comment by obetomuniz** [comment]
I'm not declaring on this way @oscar-b:

```javascript
import React from 'react';
import { injectGlobal } from 'styled-components';

injectGlobal`
  // global styles (i.e. Fonts/Resets)
`;

const Root = ({ store, history }) => (
  <div>
    // my app
  </div>
);

export default Root;
```

**Comment by oscar-b** [comment]
Try calling it in a lifecycle method which is being called on SSR (not many are, can't remember which ones right now, on my phone).

**Comment by obetomuniz** [comment]
I tried using `componentDidMount` and `componentWillMount`, but did not work. I did tests using the methods `const styles = styleSheet.getCSS();`and `styleSheet.rules().map(rule => rule.cssText).join('\n')`.

Let me know if I did it right. 

Anyway, I'll test a new approach. Instead of rendering the HTML document structure from a template file (.hbs). I'll transform this in a React Component, and if it works, probably is because`injectGlobal` has a bug and don't work for SSR.
